.github/workflows/zeebe-release.yml

```diff
@@ -244,6 +244,71 @@ jobs:
           fi
           shopt -u nocasematch # reset it
           echo "result=${PRE_RELEASE}" >> $GITHUB_OUTPUT
+      - uses: actions/checkout@v4
+        with:
+          ref: ${{ env.RELEASE_BRANCH }}
+          fetch-depth: 0 # To fetch tags as well - necessary for the changelog generation
```

Comment:
🔧 This is a costly checkout for us (with almost 40k commits on `main` alone). We could try using `fetchTags: true` instead. Just an optimization though.

Reply:
<img width="796" alt="Screenshot 2024-05-27 at 16 54 07" src="https://github.com/camunda/camunda/assets/3511026/61517d02-84a9-40e9-8f21-26f7a6392782">

Reply:
Tha KS for the proposal. In general it makes sense.

But I  don't think that it works as it expected with the proposed solution https://github.com/actions/checkout/issues/1471

I would keep it for now like it is. I think the impact it also quite small as we don't run this workflow so often.

Feel free to improve on a later stage of course :+1: